### PR TITLE
Ecommerce trends Analysis

### PR DESCRIPTION
**Pull request by adi271001**
# Pull Request for ML-Crate 💡

## Issue Title:  #652 

- **Info about the related issue (Aim of the project)** :  to analyze ecommer trends data using ml algorithms
- **Name:**  Aditya D
- **Email ID for further communication**:  ibetanibex@gmail.com
- **GitHub ID:**  https://www.github.com/adi271001
- **Idenitfy yourself:**  VSOC Contributor

Closes: #issue number that will be closed through this PR #652

### Describe the add-ons or changes you've made 📃

Give a clear description of what have you added or modifications made
Added folders containing models , images , results , requirements.txt 

## Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested? ⚙️

Describe how it has been tested
divided dataset into test and train , evaluated for test set

Describe how have you verified the changes made
i have verified the model for overfitting and underfitting via graphs

## Checklist: ☑️

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Comment by abhisheks008** [comment]
While writing the Conclusion use bullet points, otherwise it's not looking that much eye-pleasing.

**Comment by adi271001** [comment]
@abhisheks008 done please do check

**Review by abhisheks008 — APPROVED** [review]
Approved @adi271001

**Comment by abhisheks008** [comment]
While assigning your points in the VSOC, you email is not found there. @adi271001

**Comment by abhisheks008** [comment]
> While assigning your points in the VSOC, you email is not found there. @adi271001

Add your details in the Contributor-list channel in the VSOC server, otherwise your points will not going to be calculated from their end.

**Comment by adi271001** [comment]
@abhisheks008 i have added the details in contributor list please inform me once points are added

**Comment by abhisheks008** [comment]
> @abhisheks008 i have added the details in contributor list please inform me once points are added

Added your points with respect to your email ID.
